docs/CMakeLists.txt

```diff
@@ -36,17 +36,21 @@ add_custom_target(pythonpath_create
 	COMMAND ${CMAKE_COMMAND} -E make_directory ${PYTHONPATH}
 )
 
-# setuptools
-# https://pypi.python.org/pypi/setuptools
-set(SETUPTOOLS_VERSION 1.1.6)
-ExternalProject_Add(setuptools
-	DEPENDS pythonpath_create
-    URL ${CMAKE_SOURCE_DIR}/ext/setuptools-${SETUPTOOLS_VERSION}.tar.gz
-    BUILD_IN_SOURCE 1
-	CONFIGURE_COMMAND ""
-    BUILD_COMMAND ${PYTHON_CMD} setup.py build
-    INSTALL_COMMAND ${PYTHON_CMD} setup.py install --prefix=${EXTDIST_ROOT}
-)
+if(USE_EXTERNAL_SETUPTOOLS)
+    add_custom_target(setuptools /usr/bin/true)
```

Comment:
I would have expected some code to find the installed setuptools Python library.